sv2/channels-sv2/src/server/standard.rs

```diff
@@ -582,29 +617,29 @@ where
         };
 
         // convert the header hash to a target type for easy comparison
-        let hash = header.block_hash();
-        let raw_hash: [u8; 32] = *hash.to_raw_hash().as_ref();
-        let block_hash_target = Target::from_le_bytes(raw_hash);
-        let hash_as_diff = block_hash_target.difficulty_float();
+        let share_hash = header.block_hash();
+        let share_raw_hash: [u8; 32] = *share_hash.to_raw_hash().as_ref();
+        let share_hash_target = Target::from_le_bytes(share_raw_hash);
+        let share_hash_as_diff = share_hash_target.difficulty_float();
         let network_target = Target::from_compact(nbits);
 
         // print hash_as_target and self.target as human readable hex
-        let block_hash_target_bytes = block_hash_target.to_be_bytes();
-        let target_bytes = self.target.to_be_bytes();
+        let share_hash_target_bytes = share_hash_target.to_be_bytes();
+        let job_target_bytes = self.target.to_be_bytes();
```

Comment:
I'm pretty sure the issue which I reported on https://github.com/stratum-mining/stratum/pull/2000#issuecomment-3665054150  is due to the fact that here we are still getting the channel's target instead of the job one.


```suggestion
        let job_target_bytes = job_target.to_be_bytes();
```